spec/integration_spec.rb

```diff
@@ -70,6 +70,17 @@
     end
   end
 
+  context "PDF with a difference table (v2)" do
+    let(:filename) { pdf_spec_file("difference_table2") }
+
+    it "extracts text correctly" do
+      PDF::Reader.open(filename) do |reader|
+        page = reader.page(1)
+        expect(page.text).to eql("This PDF contains ligatures,for example in “ﬁle”and “ﬂoor”.")
```

Comment:
This isn't perfect - it'd be nice to convert the `ﬁ` and `ﬂ` ligatures to `fi` and `fl`. Still this is better than what we had.